inksdk/ink/src/main/java/com/microsoft/device/ink/InkView.kt

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ */
+
+package com.microsoft.device.ink
+
+import android.content.Context
+import android.graphics.*
```

Comment:
ktlint will complain if it has the wildcard character since it's a good practice to import jus the classes you are using.